workshop/exercise-8/README.md

```diff
@@ -1,4 +1,4 @@
-# Exercise 8 - Policy Enforcement 
+# Exercise 8 - Policy Enforcement
 
 Backend systems such as access control systems, telemetry capturing systems, quota enforcement systems, billing systems, and so forth, traditionally directly integrate with Services, creating a hard coupling and baking-in specific semantics and usage options.
 
```

Comment:
The `mixer-rule-denial.yaml` rule does not exist in the `guestbook` or the `istio101` files. This is a rule from the bookinfo sample app set, but is not define for the guestbook sample set.

Reply:
@szihai Please review the above issue, raised by a tester.